docs/deployment/heroku.md

```diff
@@ -5,69 +5,102 @@ sidebar_label: Heroku
 description: How to deploy Backstage to Heroku
 ---
 
-Heroku is a Platform as a Service (PaaS) designed to handle application
-deployment in a hands-off way. Heroku supports container deployment of Docker
-images, a natural fit for Backstage.
+Heroku is a Platform as a Service (PaaS) designed to simplify application deployment.
 
-## Configuring the CLI
+## Create App
 
-First, install the
-[heroku-cli](https://devcenter.heroku.com/articles/heroku-cli) and login:
+Start with an exisiting Backstage app or follow the [getting started guide](https://backstage.io/docs/getting-started/) to create a new one:
 
 ```shell
-$ heroku login
+$ npx @backstage/create-app@latest
+  Enter a name <your-app>
 ```
 
-If you have not yet created a project through the Heroku interface, you can create it through the CLI.
+Install the
+[Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and create a new Heroku app:
 
 ```shell
-$ heroku create <your-app>
+$ cd your-app/
+$ heroku apps:create <your-app>
 ```
 
-You _might_ also need to set your Heroku app's stack to `container`:
+## Domain
 
-```bash
-$ heroku stack:set container -a <your-app>
+Get Heroku app URL:
+```shell
+$ heroku domains -a <your-app>
+<your-app-123>.herokuapp.com
 ```
 
-Configuring your `app-config.yaml`:
-
+Update `baseUrl` and `port` in `app-config.yaml`. The core [app-backend plugin](https://www.npmjs.com/package/@backstage/plugin-app-backend) allows a single Heroku app to serve the frontend and backend:
 ```yaml
 app:
-  # Should be the same as backend.baseUrl when using the `app-backend` plugin
-  baseUrl: https://<your-app>.herokuapp.com
+  baseUrl: https://<your-app-123>.herokuapp.com
 
 backend:
-  baseUrl: https://<your-app>.herokuapp.com
+  baseUrl: https://<your-app-123>.herokuapp.com
   listen:
     port:
       $env: PORT
       # The $PORT environment variable is a feature of Heroku
       # https://devcenter.heroku.com/articles/dynos#web-dynos
 ```
 
-> Make sure your file is being copied into your container in the `Dockerfile`.
+## Build Script
+
+Add a build script in `package.json` to compile frontend during deployment:
+```json
+"scripts": {
+  "build": "yarn build:backend --config ../../app-config.yaml"
+```
+
+## Start Command
```

Comment:
Is there a reason we aren't suggesting running in a Docker container? We do this right now with the Demo site. I think perhaps we should keep both methods.

Reply:
I have added the docker instructions.